cmd/cpe/generate.go

```diff
@@ -56,25 +82,51 @@ func main() {
 	err = vulnerabilities.GenerateCPEDB(dbPath, cpeDict)
```

Comment:
Eventually, (if things start to take a bit longer to generate) we should probably create the sqlite db at this level and then spawn goroutines to generate all the different tables in parallel.